db/structure.sql

```diff
@@ -1378,3 +1373,5 @@ INSERT INTO "schema_migrations" (version) VALUES
 ('20230401114456'),
 ('20231120164231'),
 ('20231120164346');
+
+
```

Comment:
Yeah, not sure why this happens, probably due to the different PostgreSQL versions we used during the migration to the new infra... Maybe we can revert this for now? I think we should solve it separately.

Reply:
Yes, could you do it?

Reply:
Could you please 🙏🏼 restore it by coping the file from develop branch?

Reply:
Done